common/js/options.js

```diff
@@ -12,19 +12,22 @@ var refresh_button;
 var default_button;
 var logout_button;
 var save_button;
-
+var preview;
+var form;
 var forceSecure;
 var httpWarning_label;
 
 var bodyWidth;
 var bodyHeight;
 var historyNum;
+var bodyWidthNum = min_width;
+var bodyHeightNum = min_height;
 
 function secureWarning() {
     if (forceSecure.checked) {
-        httpWarning_label.style.display = "none";
+        httpWarning_label.style.opacity = 0;
```

Comment:
JS now changes opacity 0/1 instead of display none/block because opacity is transitionable and looks less janky.